### PR TITLE
add method to parse field definition from Parser

### PR DESCRIPTION
**Pull request by andimarek**

### CODE REVIEW

File: src/test/groovy/graphql/parser/ParserTest.groovy [hunk 1]
```diff
@@ -964,6 +961,22 @@ triple3 : """edge cases \\""" "" " \\"" \\" edge cases"""
...
+        AstPrinter.printAstCompact(fieldDefinition) == valueLiteral
+
+        where:
+        valueLiteral                                              | _
+        'foo: Foo'                                                | _
+        'foo(a:String): Foo'                                      | _
+        'foo(a:String!,b:Int!): Foo'                              | _
+        'foo(a:String! ="defaultValue",b:Int!): Foo'              | _
+        'foo(a:String!,b:Int!): Foo @directive(someValue:String)' | _
+    }
```
Comment (bbakerman) [inline src/test/groovy/graphql/parser/ParserTest.groovy:977]: What about the failure case - what happens when it fails to parse?  Will it throw and exception and if so lets lock that in with a test
